clients/python/main.py

```diff
@@ -1,22 +1,19 @@
 from expclient import ExperimentationClient
-from cacclient import CacClient 
+from cacclient import CacClient , MergeStrategy
 import http.server
 import socketserver
 
 try:
-    tenant_name = "dev"
+    tenant_name = "test"
     polling_frequency = 1
     cac_host_name = "http://localhost:8080"
     exp_client = ExperimentationClient(tenant_name, polling_frequency, cac_host_name)
     cac_client = CacClient(tenant_name, polling_frequency, cac_host_name)
 
-    exp_client.create_new_experimentation_client()
-    cac_client.create_new_cac_client()
-
     cac_client.start_cac_polling_update()
     exp_client.start_experimentation_polling_update()
 
-    PORT = 8002
+    PORT = 8003
```

Comment:
Okay with this, but any reason to change this?

Reply:
No, I will change